spec/supplejack_common/xml/base_spec.rb

```diff
@@ -16,6 +16,20 @@
     klass.clear_definitions
   end
 
+  describe ".record_selector" do
+    it 'assignes the record selector xpath class attributed' do
+      klass.record_selector("//o:ListRecords/o:record")
+      expect(klass._record_selector).to eq "//o:ListRecords/o:record"
+    end
+  end
+
+  describe "next_page_token" do
```

Comment:
Use of single quotes and add `.` (dot) in front of `next_page_token`.

Reply:
I've added the ., but I'm not adding single quotes when the whole repo uses doubles.  I think it's better to be consistent.

Reply:
I've drafted a story to add the rubop cop @hapiben